### ISSUE TITLE
Could I know the incorrect point of my project's structure?

### ISSUE THREAD

**Issue by chojpsh1** [azure-rtos/filex#23]
Hello, I create a  azure rtos project with below structure on FVP Cortex A55x8. And this project try to run the fx_media_format function with tx_thread_create.
azure_rtos => build the 'azrtos' elf file
ㄴ app
ㄴ core
    ㄴ inc => tx_user.h
    ㄴ src
ㄴ filex
    ㄴ inc => fx_user.h
    ㄴ src
ㄴ Middlewares
    ㄴ threadx
        ㄴ common_smp
            ㄴ inc
            ㄴ src
        ㄴ ports_smp
            ㄴ cortex_a55_smp
                ㄴ gnu
                    ㄴ inc
                    ㄴ src
    ㄴ filex
        ㄴ common
            ㄴinc
            ㄴsrc
        ㄴ ports
            ㄴ cortex_a55_smp
                ㄴ gnu
                    ㄴ inc
                    ㄴ src
But, the thread isn't running. Could I know What should I modify? I also defined the TX_ARMV8_2 and EL1.

**Comment by goldscott** [comment]
Before doing anything with FileX, have you got the basic ThreadX example running?

**Comment by chojpsh1** [comment]
Yes, I checked the ThreadX smp sample. For using FileX, I copied ports/cortex_m4/gnu/* to ports/cortex_a55_smp/gnu/*.

**Comment by chojpsh1** [comment]
I put the threadx into filex as subdirectory and it's worked well. Maybe there is something problem about linking project. Anyway, thank you for your checking! :)
